torch/types.py

```diff
@@ -63,6 +64,8 @@
 # Meta-type for "numeric" things; matches our docs
 Number: TypeAlias = Union[int, float, bool]
 
+FileLike: TypeAlias = Union[str, os.PathLike[str], IO[bytes]]
```

Comment:
Nit: is the FileLIke really the best name? It's more FilePath like right? (can't call write on a str()) after all.

Reply:
These stubs also have some examples of custom Protocol that could be an alternative to `IO[bytes]` in general.